Policies/github-corporate-terms-of-service.md

```diff
@@ -164,7 +164,7 @@ You retain all moral rights to Your Content that you or your Users upload, publi
 To the extent this agreement is not enforceable by applicable law, you grant GitHub the rights we need to use Your Content without attribution and to make reasonable adaptations of Your Content as necessary to render the Website and provide the Service.
 
 ### E. Private Repositories
-**Short version:** *You may have access to private repositories. The content of private repositories is confidential, and we only access it for support reasons, with your consent, or if required for maintenance or security reasons.*
+**Short version:** *You may have access to private repositories. We treat the content of private repositories as confidential, and we only access it for support reasons, with your consent, or if required to for security reasons.*
```

Comment:
@PKI-Kent That's actually not a typo: read it as "...if required to do so, for security reasons."

...I might could rephrase that.